packages/lib/shim-init-node.js

```diff
@@ -250,9 +251,12 @@ function shimInit(options = null) {
 	// destinationResourceId option. This method is indirectly tested in
 	// Api.test.ts.
 	shim.createResourceFromPath = async function(filePath, defaultProps = null, options = null) {
-		options = { resizeLargeImages: 'always', // 'always', 'ask' or 'never'
+		options = {
```

Comment:
I'll take a look on these changes. Not consider it part of the final code yet, please.